src/style.scss

```diff
@@ -0,0 +1,424 @@
+$base: rgb(50, 50, 50);
+$background_2: rgb(46, 179, 152);
+$background: rgb(255, 255, 255);
```

Comment:
Use some understandable names: backgroundWhite / backgroundGreen / backgroundDark etc. In some time you will forget what means "background_2" and you will have to check the color before using it. So, select the correct name to avoid unnecessary actions in future.